docs/Configuration.md

```diff
@@ -233,9 +233,15 @@ locale = da_DK en_US fi_FI fr_FR nb_NO sv_SE
 If a new `locale tag` is added to the configuration then the equivalent
 MO file should be added to Zonemaster-Engine at the correct place so
 that gettext can retrieve it, or else the added `locale tag` will not
-add any actual language support. See the
-[Zonemaster-Engine share directory] for the existing PO files that are
-converted to MO files. (Here we should have a link
+add any actual language support. The MO file should be created for the
+`language code` of the `locale tag` (see the table above), not the entire
+`locale tag`. E.g. if the `locale` configuration key includes "sv_SE" then
+a MO file for "sv" should be included in the installation.
+
+Use of MO files based on the entire `locale tag` is *deprecated*.
```

Comment:
What's the reason for deprecating MO files with country code in their file names? We're not dropping country codes in the configuration file. I think it would make sense to keep the naming of the MO files consistent with the configuration format.

Reply:
If the MO file is based on, say, "fr_FR", it will not work with "fr_CA". If `LANGUAGE` is set to "fr_CA.UTF-8" `zonemaster-cli` will do the proper translation if the MO file is based in "fr" rather than "fr_FR". Since the translation is made country neutral there is no reason to base the MO files on the complete `locale tag`. This is no change from today. It is just that it is clarified. No value is added by basing the MO file on, say, "fr_FR".